.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!-- ************************** IMPORTANT ************************* -->
+<!-- ** Continuous Deployment is enabled for this repository     ** -->
+<!-- ** Merging into master = deploying to PROD                  ** -->
+<!-- **                                                          ** -->
+<!-- ** Use Riff-Raff to deploy/test a PR on CODE before merging ** -->
```

Comment:
Is it worth clarifying that this is only if you need to test your changes in a production "like" environment?